ui/pages/confirmations/components/confirm/header/wallet-initiated-header.tsx

```diff
@@ -38,7 +41,26 @@ export const WalletInitiatedHeader = () => {
   const handleBackButtonClick = useCallback(async () => {
     const { id } = currentConfirmation;
 
-    await dispatch(editExistingTransaction(AssetType.token, id.toString()));
+    const isNativeSend =
+      currentConfirmation.type === TransactionType.simpleSend;
+    const isERC20TokenSend =
+      currentConfirmation.type === TransactionType.tokenMethodTransfer;
+    const isNFTTokenSend =
```

Comment:
Minor, if these booleans only exist to support the if below, would it be cleaner to use a `switch`:

```
let assetType: AssetType;

switch(currentConfirmation.type) {
    case TransactionType.simpleSend:
        assetType = AssetType.native;
        break;

    ...
```